components/list/list-item-drag-handle.js

```diff
@@ -266,49 +272,19 @@ class ListItemDragHandle extends LocalizeCoreElement(RtlMixin(LitElement)) {
 			default:
 				return;
 		}
-		this._dispatchAction(action);
-		e.preventDefault();
-		e.stopPropagation();
-		const cell = findComposedAncestor(this, (parent) =>  parent.hasAttribute && parent.hasAttribute('draggable'));
-		if (cell) await cell.updateComplete;
-		await this.updateComplete;
-		this.focus();
-		this._movingElement = false;
-	}
 
-	_onFocusInKeyboardButton() {
-		if (hasDisplayedKeyboardTooltip) return;
-		this._displayKeyboardTooltip = true;
-		hasDisplayedKeyboardTooltip = true;
-	}
-
-	_onFocusOutKeyboardButton(e) {
-		this._displayKeyboardTooltip = false;
-		if (this._movingElement) {
-			this._movingElement = false;
-			e.stopPropagation();
-			e.preventDefault();
-			return;
-		}
-		this._keyboardActive = false;
-		this._dispatchAction(dragActions.save);
+		e.preventDefault();
 		e.stopPropagation();
-	}
+		this._doAction(action);
 
-	_onInactiveKeyboard(e) {
```

Comment:
This just got split up into `_onDraggerButtonClick` and `_onDraggerButtonKeydown` when `_onInactiveKeyboard` was renamed for clarity.